modules/token-overview.adoc

```diff
@@ -0,0 +1,66 @@
+
+The following sections provide more tails about the token types available with {productname}.
+
+[id="token-types"]
+== {productname} token types
+
+{productname} primarily supports two types of tokens: OAuth2 access tokens and robot account tokens. A third token type, an _OCI referrers access token_, that is required to list OCI referrers of a manifest under a repository, is also available when warranted.
+
+[discrete]
+[id="oauth2-access-tokens"]
+=== OAuth2 access tokens 
+
+OAuth 2 access tokens enable user-authenticated access to the {productname} API, suitable for applications that require user identity verification. These tokens are obtained through an OAuth2 authorization process, where a {productname} administrator generates a token on behalf of themselves or another user to access {productname} resources. These tokens provide scope-based access and specify what resources and actions the token holder is allowed. Currently, the following options can be selected for a token holder:
+
+* *Administer Organization*. When selected, allows the user to be able to administer organizations, including creating robots, creating teams, adjusting team membership, and changing billing settings.
+
+* *Administer Repositories*. When selected, provides the user administrator access to all repositories to which the granting user or robot account has access.
+
+* *Create Repositories*.  When selected, provides the user the ability to create repositories in any namespaces that the granting user or robot account is allowed to create repositories. 
+
+* *View all visible repositories*. When selected, provides the user the ability to view and pull all repositories visible to the granting user or robot account.
+
+* *Read/Write to any accessible repositories*.  When selected, provides the user the ability to view, push and pull to all repositories to which the granting user or robot account has write access.
+
+* *Super User Access*.  When selected, provides the user the ability to administer your installation including managing users, managing organizations and other features found in the superuser panel. 
+
+* *Administer User*  When selected, provides the user the ability to  administer your account including creating robots and granting them permissions to your repositories. 
+
+* *Read User Information*.  When selected, provides the user the ability to read user information such as username and email address.
+
+Token distributors should be mindful of the permissions that they are granting when generating a token on behalf of a user, and should have absolute trust in a user before granting such permissions as *Administer organization*, *Super User Access*, and *Administer User*.
+
+OAuth2 access tokens are passed as a `Bearer` token in the `Authorization` header of an API call; this means that they can be used to directly access a {productname} resource, such as an image tag, a repository, an organization, and so on.
```

Comment:
Not sure I understand this sentence. Can you explain what you meant? Access to a specific resource is done via API endpoints, the bearer token is just there to provide authentication and authorization.

Reply:
fixed.